tracing/trace_keyspace_helper.cc

```diff
@@ -290,9 +290,9 @@ cql3::query_options trace_keyspace_helper::make_slow_query_mutation_data(const o
     auto millis_since_epoch = std::chrono::duration_cast<std::chrono::milliseconds>(record.started_at.time_since_epoch()).count();
 
     // query command is stored on a parameters map with a 'query' key
-    auto query_str_it = record.parameters.find("query");
+    const auto query_str_it = record.parameters.find("query");
     if (query_str_it == record.parameters.end()) {
-        throw std::logic_error("No \"query\" parameter set for a session requesting a slow_query_log record");
+        tlogger.warn("No \"query\" parameter set for a session requesting a slow_query_log record");
```

Comment:
Do I understand correctly that this warning will also be printed for correct batch statements? In that case it's redundant, so perhaps it's better to remove it altogether or degrade to debug/trace